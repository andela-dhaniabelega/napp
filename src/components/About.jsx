import React from 'react';
import '../../public/sass/styles.scss';
import Header from '../components/Header';
import AboutSlider from '../components/AboutSlider';
import Footer from '../components/Footer';

/**
 * Stateless component
 * Simply renders information
 */
const About = () => (
  <div className="wrapper">
    <div className="center">
      <Header />
      <AboutSlider />
      <div className="about-text">
        <div className="row">
          <p className="col-md-3">
            <strong>LEADERSHIP</strong><br />
            <strong>Chief Executive Officer</strong>: Ken Adams<br />
            <strong>Editor-in-Chief</strong>: Regina Philange<br />
            <strong>Head of Product</strong>: Julia Roberts<br />
            <strong>Head of Video</strong>: Gunther Hayes<br />
            <strong>Chief of Staff</strong>: Phoebe Buffay<br />
            <strong>Head of Business Development</strong>: Mark Edom<br />
            

          </p>
          <p className="col-md-3">
            <strong>EDITORIAL</strong><br />
            <strong>Editorial Director, Lifestyle</strong>: Ross Roberts<br />
            <strong>News Director</strong>: Emeka Nwankwo<br />
            <strong>
            Editorial Director, News & Product
            </strong>: Whitney Hoyt<br />
            <strong>Director of Growth & Analytics</strong>: Memuna Haruna<br />
            <strong>Global Editorial Director</strong>: Audax Achirinah<br />
            <strong>DC Bureau Chief</strong>: JohnBosco Onyia<br />
          </p>
          <p className="col-md-3">
            <strong>DISTRIBUTION</strong><br />
            <strong>Global Social Media Editor</strong>: Ethan Klapper<br />
            <strong>Social Media Editors</strong>: Mayowa Makinde<br />
            <strong>
            Associate Social Media Editor
            </strong>: Adebayo Simulator<br />
            <strong>
             Associate Editors, News & Emerging Platforms
            </strong>: Kenny SuccessTeam

          </p>

          <p className="col-md-3">
            <strong>NEWSDESK</strong><br />
            <strong>Managing Editor</strong>: Robyn Malcolm<br />
            <strong>Senior Editor</strong>: Ani Vrabel<br />
            <strong>News Editors</strong>: Alexander Eichler, Aurora Ellis<br />
            <strong>Mari Hayman, Kurt Heine, Grace Maalouf, Bobbi Olson</strong>
          </p>
        </div>
      </div>
      <Footer />
    </div>
  </div>
);

export default About;
