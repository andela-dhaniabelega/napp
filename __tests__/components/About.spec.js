import React from 'react';
import renderer from 'react-test-renderer'
import About from '../../src/components/About';


describe('About Page', () => {
		it('renders a snapshot', () => {
          const tree = renderer.create(<About />).toJSON();
          expect(tree).toMatchSnapshot();
		});
});