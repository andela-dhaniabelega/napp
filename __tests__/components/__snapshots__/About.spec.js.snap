// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page renders a snapshot 1`] = `
<div
  className="wrapper"
>
  <div
    className="center"
  >
    <div
      className="header"
    >
      <div
        className="logo floatleft"
      >
        <a
          href="/"
        >
          <img
            alt=""
            src="../public/images/logo copy.png"
          />
        </a>
      </div>
      <div
        className="top_menu floatleft"
      >
        <ul>
          <li>
            <a
              href="/"
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/about"
            >
              About Napp
            </a>
          </li>
          <li>
            <a
              href="/contact"
            >
              Contact us
            </a>
          </li>
          <li>
            <a
              onClick={[Function]}
              role="button"
            >
              Sign In with Google
            </a>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="slider-margin"
    >
      <div
        className="main_menu_area"
      >
        <ul
          id="nav"
        />
      </div>
      <div
        className="slider_area"
      >
        <div
          className="slider"
        >
          <ul
            className="bxslider"
          >
            <li>
              <img
                alt=""
                src="../public/images/about.jpg"
              />
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="about-text"
    >
      <div
        className="row"
      >
        <p
          className="col-md-3"
        >
          <strong>
            LEADERSHIP
          </strong>
          <br />
          <strong>
            Chief Executive Officer
          </strong>
          : Ken Adams
          <br />
          <strong>
            Editor-in-Chief
          </strong>
          : Regina Philange
          <br />
          <strong>
            Head of Product
          </strong>
          : Julia Roberts
          <br />
          <strong>
            Head of Video
          </strong>
          : Gunther Hayes
          <br />
          <strong>
            Chief of Staff
          </strong>
          : Phoebe Buffay
          <br />
          <strong>
            Head of Business Development
          </strong>
          : Mark Edom
          <br />
        </p>
        <p
          className="col-md-3"
        >
          <strong>
            EDITORIAL
          </strong>
          <br />
          <strong>
            Editorial Director, Lifestyle
          </strong>
          : Ross Roberts
          <br />
          <strong>
            News Director
          </strong>
          : Emeka Nwankwo
          <br />
          <strong>
            Editorial Director, News & Product
          </strong>
          : Whitney Hoyt
          <br />
          <strong>
            Director of Growth & Analytics
          </strong>
          : Memuna Haruna
          <br />
          <strong>
            Global Editorial Director
          </strong>
          : Audax Achirinah
          <br />
          <strong>
            DC Bureau Chief
          </strong>
          : JohnBosco Onyia
          <br />
        </p>
        <p
          className="col-md-3"
        >
          <strong>
            DISTRIBUTION
          </strong>
          <br />
          <strong>
            Global Social Media Editor
          </strong>
          : Ethan Klapper
          <br />
          <strong>
            Social Media Editors
          </strong>
          : Mayowa Makinde
          <br />
          <strong>
            Associate Social Media Editor
          </strong>
          : Adebayo Simulator
          <br />
          <strong>
            Associate Editors, News & Emerging Platforms
          </strong>
          : Kenny SuccessTeam
        </p>
        <p
          className="col-md-3"
        >
          <strong>
            NEWSDESK
          </strong>
          <br />
          <strong>
            Managing Editor
          </strong>
          : Robyn Malcolm
          <br />
          <strong>
            Senior Editor
          </strong>
          : Ani Vrabel
          <br />
          <strong>
            News Editors
          </strong>
          : Alexander Eichler, Aurora Ellis
          <br />
          <strong>
            Mari Hayman, Kurt Heine, Grace Maalouf, Bobbi Olson
          </strong>
        </p>
      </div>
    </div>
    <div
      className="footer_bottom_area"
    >
      <div
        className="copyright_text"
      >
        <p>
          Copyright Â© 2017 Napp. All rights reserved
        </p>
      </div>
    </div>
  </div>
</div>
`;
